MIGRATION.md

```diff
@@ -339,6 +340,26 @@
 
 ## From version 7.x to 8.0.0
 
+### Manager addons are now rendered with react 18
+
+The UI added to the manager via addon, are now rendered with react 18.
```

Comment:
@ndelangen when you have a moment can you adjust this sentence to be: 
"The UI added to the manager via addons is now rendered with React 18."

Also, can you capitalize react in the heading and adjust the link above accordingly?